src/AddTodoForm.js

```diff
@@ -15,7 +18,7 @@ const AddTodoForm = ({ onAddTodo }) => {
   const handleAddTodo = (addToDoEvent) => {
     // prevent form from refreshing
     addToDoEvent.preventDefault();
-    // Instructions from 1-4 for below: "Inside `handleAddTodo`, update the `onAddTodo` callback prop to pass an Object instead of a String; Object should have the following properties:
+    // Lesson 1-4 instructions for below: "Inside `handleAddTodo`, update the `onAddTodo` callback prop to pass an Object instead of a String; Object should have the following properties:
 
     // 1) 'title`: equal to `todoTitle (or 'todoTitleValue' in our case)`;
```

Comment:
Remove all the comments. There is no need declaring all the procedures once again in your file.